app/admin/database/migrations/2021_11_28_000300_create_stocks_table.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Admin\Database\Migrations;
+
+use Admin\Models\Stocks_model;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStocksTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('stocks', function (Blueprint $table) {
+            $table->engine = 'InnoDB';
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('location_id');
+            $table->unsignedBigInteger('stockable_id');
+            $table->string('stockable_type');
+            $table->bigInteger('quantity')->nullable();
+            $table->boolean('low_stock_alert')->default(0);
+            $table->integer('low_stock_threshold')->default(0);
+            $table->boolean('is_tracked')->default(0);
+            $table->timestamps();
+        });
+
+        Schema::create('stock_history', function (Blueprint $table) {
+            $table->engine = 'InnoDB';
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('stock_id');
+            $table->unsignedBigInteger('staff_id')->nullable();
+            $table->unsignedBigInteger('order_id')->nullable();
+            $table->string('state');
+            $table->bigInteger('quantity');
+            $table->timestamps();
+
+            $table->foreign('stock_id')->references('id')->on('stocks')->cascadeOnDelete()->cascadeOnUpdate();
+            $table->foreign('order_id')->references('order_id')->on('orders')->cascadeOnDelete()->cascadeOnUpdate();
+        });
+
+        $this->copyStockQtyFromMenus();
+
+        $this->copyStockQtyFromMenuOptions();
+
+        Schema::table('menus', function (Blueprint $table) {
+            $table->dropColumn('stock_qty');
```

Comment:
On a fresh install I'm getting `SQLSTATE[42S22]: Column not found: 1054 Unknown column 'stock_qty' in 'field list' ` due to this change, so the seed data needs to be updated too